### ISSUE TITLE
Pretrained

### ISSUE THREAD

**Issue by Ella77** [valohai/deep-fashion-detection#1]
Thank you for great repository 
Is there any available pretrained model??

**Comment by sgflower66** [comment]
I have same question~

**Comment by ruksi** [comment]
no, we don't have a pretrained model anywhere publicly available, I could do it at some point ;)
